INSTALL

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh
 
+web_get() {
+	wget -q
+}
+
```

Comment:
1st problem

INSTALL not check dependencies, will try run installation even without curl/wget

Reply:
you need to add "$@" to expand the arguments, as I did at https://github.com/ivan-hc/AM/pull/551/commits/a05f4bc34cacbc49738446995f17aa61d504f88d